tests/auth_tests/test_tokens.py

```diff
@@ -128,29 +115,34 @@ def test_token_with_different_secret(self):
         self.assertIs(p0.check_token(user, tk0), True)
         # Create and check a token with the default secret.
         p1 = PasswordResetTokenGenerator()
-        self.assertEqual(p1.secret, settings.SECRET_KEY)
-        self.assertNotEqual(p1.secret, new_secret)
         tk1 = p1.make_token(user)
         # Tokens created with a different secret don't validate.
-        self.assertIs(p0.check_token(user, tk1), False)
-        self.assertIs(p1.check_token(user, tk0), False)
+        self.assertFalse(p0.check_token(user, tk1))
```

Comment:
Again in this file. 

`assertIs()` is preferred style. https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style